### ISSUE TITLE
[Lattice] Can't open .cube LUTs - macOS Sequoia 15.1

### ISSUE THREAD

**Issue by cameronrad** [videovillage/community#202]
Ever since updating to macOS Sequoia I've been unable to directly open .cube LUT files in Lattice. 

Trying to open them directly in Lattice throws an error stating:
> The document “Test.cube” could not be opened. Lattice cannot open files in the “MLUT_File” format.

However if I create a new LUT in Lattice, I can drag the .cube LUT into the application window and it works. 

The quicklook extension also doesn't seem to be working. I've deleted and reinstalled the application a few times and it hasn't fixed anything. I might need to try deleting some of the application support folders as well.

**Comment by gregcotten** [comment]
I'm not 100% sure but this isn't happening to me on 15.1 - are you sure you haven't installed another app that might try to interfere with file type associations? I think in the past a DJI app has tried to re-associate the cube file format as something different...

**Comment by cameronrad** [comment]
Hmmm weird!! I don't think I have installed anything that'd interfere. I don't have many video related applications installed, I mainly do stills work. I have Resolve, Screen, Filmbox, Lattice, LUTCalc and a few others, but nothing DJI. related. It's happening on two of my systems. A Mac Studio and Macbook Pro. 

I also set Lattice to be the default app to open .cube files but whenever I try to open them within the app, I get the  "Lattice cannot open files in the “MLUT_File” format." error. If I go File > Open LUT it also throws the error and .cube files are grayed out by default. No issue if I drag the .cube file into app window however. 

Are there any files in ~/Library/ I should try removing and reinstalling?

**Comment by cameronrad** [comment]
Fixed it! 

Had to rebuild the macOS Launch Services database by pasting this in Terminal:

`/System/Library/Frameworks/CoreServices.framework/Frameworks/LaunchServices.framework/Support/lsregister -kill -r domain local -domain system -domain user`

Source: https://helpx.adobe.com/photoshop/kb/error-filenamepsd-could-opened-occurs.html

**Comment by cameronrad** [comment]
Update on this. 

So the problem came back and I ended up tracking it down to an obscure source, an iOS app. macOS Sequoia added iPhone mirroring and an app from my iPhone ended up causing some file association conflicts on my desktop. The app is a Sony app called Monitor & Control. 

I noticed the problem would temporarily go away if I ran the above code in Terminal, however running that would cause other issues like System Preferences not working correctly. Also the problem would just return after reboot. 

I ended up running 
`/System/Library/Frameworks/CoreServices.framework/Frameworks/LaunchServices.framework/Support/lsregister -dump`

I searched for instances of "MLUT_File" and noticed it was associated with the iOS app and that a copy of the app was located in ~/Library/Daemon Containers. I deleted the app from there and from my phone. So far the problem seems solved. 

This what I was able to find about the Daemon Containers folder https://eclecticlight.co/2024/08/05/what-are-all-those-containers/
